contracts/DAORegistry.sol

```diff
@@ -1,19 +1,16 @@
 pragma solidity ^0.5.4;
 
-import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "@openzeppelin/upgrades/contracts/Initializable.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/ownership/Ownable.sol";
 
-contract DAORegistry is Ownable {
+contract DAORegistry is Initializable, Ownable {
```

Comment:
Initializable should be removed.
it is already part of Ownable

Reply:
done